### PR TITLE
Develop 2.0

### PR DESCRIPTION
**Pull request by shreyaagrawal0809**

### CODE REVIEW

File: assam_budget_board/variables.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+be  =  "Budget 2019-20"
```
Comment (gggodhwani) [inline assam_budget_board/variables.py]: Can we move these to settings.py? Also, can we use the variables formats as in settings.py as the standard nomenclature. Also, "_" and CamelCase don't go together.

File: assam_budget_board/cms_plugins.py [hunk 2]
```diff
@@ -18,15 +18,15 @@ def render(self, context, instance, placeholder):
...
+                    "be"  : "Budget %s-%s" % (settings.FISCAL_YEAR, (settings.FISCAL_YEAR+1)%1000),
+                    "actuals" : "Actuals %s-%s"% (settings.FISCAL_YEAR-2, (settings.FISCAL_YEAR-1)%1000),
+                    "bePrev" : "Budget %s-%s"% (settings.FISCAL_YEAR-1, (settings.FISCAL_YEAR)%1000),
+                    "re" : "Revised %s-%s" %  (settings.FISCAL_YEAR-1, (settings.FISCAL_YEAR)%1000)
         		},
         		'headHierarchy' : headHierarchy,
         		'tableColumns' : tableColumns,
         	})
-
+        print("Budget %s-%s" % (settings.FISCAL_YEAR, (settings.FISCAL_YEAR+1)%1000))
```
Comment (gggodhwani) [inline assam_budget_board/cms_plugins.py]: @shreyaagrawal0809  Can we remove print from here?

File: assam_budget_board/cms_plugins.py [hunk 3]
```diff
@@ -41,13 +41,11 @@ def render(self, context, instance, placeholder):
...
-        		"be" : "Budget 2018-19", 
-        		"actuals" : "Actuals 2016-17",
-				"bePrev" : "Budget 2017-18",
-				"re" : "Revised 2017-18"
+                "be" : "BUDGET_%s_%s" % (settings.FISCAL_YEAR, (settings.FISCAL_YEAR+1)%1000), 
+                "bePrev" : "BUDGET_%s_%s" % (settings.FISCAL_YEAR-1, (settings.FISCAL_YEAR)%1000)
         		}
         	})
-
+        print("BUDGET_%s_%s" % (settings.FISCAL_YEAR, (settings.FISCAL_YEAR+1)%1000))
```
Comment (gggodhwani) [inline assam_budget_board/cms_plugins.py]: @shreyaagrawal0809 Can we remove print from here?

File: assam_budget_board/variables.py (hunk 1 above)
Comment (gggodhwani) [reply assam_budget_board/variables.py]: @shreyaagrawal0809 Do we still need this file? Otherwise git rm this file

Comment (shreyaagrawal0809) [reply assam_budget_board/variables.py]: Done

File: assam_budget_board/cms_plugins.py (hunk 2 above)
Comment (shreyaagrawal0809) [reply assam_budget_board/cms_plugins.py]: Done

File: assam_budget_board/cms_plugins.py (hunk 3 above)
Comment (shreyaagrawal0809) [reply assam_budget_board/cms_plugins.py]: Done

**Comment by gggodhwani** [comment]
Looks good to me :+1:

**Pull request merged**